src/libdrakvuf/linux-processes.c

```diff
@@ -267,3 +267,68 @@ bool linux_get_current_thread_id( drakvuf_t drakvuf, uint64_t vcpu_id, uint32_t
 
     return true;
 }
+
+vmi_pid_t linux_get_process_ppid( drakvuf_t drakvuf, addr_t process_base )
```

Comment:
I think this function would be more robust if it returned `vmi_status_t` and the ppid through a passed-in pointer.

Reply:
It's just mimicing how the ```xxx_get_process_pid()``` (linux and windows) are currently defined

Reply:
I'm compiling with debug enabled by the way...

Reply:
Yea, maybe those should be changed too =)

Reply:
Do you want me to change also ```win_get_process_ppid``` or just making a function wrapper that will be called from ```linux_get_process_ppid```?

Reply:
Ok, so I will change the 4 functions and the corresponding ```drakvuf_get_process_pid``` and ```drakvuf_get_process_ppid``` API definitions. 👍